templates/repo/branch/list.tmpl

```diff
@@ -43,7 +43,7 @@
 									<a role="button" class="btn interact-bg gt-p-3" href="{{$.FeedURL}}/rss/branch/{{PathEscapeSegments .DefaultBranchBranch.DBBranch.Name}}">{{svg "octicon-rss"}}</a>
 								{{end}}
 								{{if not $.DisableDownloadSourceArchives}}
-									<div class="ui dropdown btn interact-bg gt-p-3" data-tooltip-content="{{$.locale.Tr "repo.branch.download" ($.DefaultBranchBranch.DBBranch.Name)}}">
+									<div class="ui dropdown btn interact-bg branch-download" data-tooltip-content="{{$.locale.Tr "repo.branch.download" ($.DefaultBranchBranch.DBBranch.Name)}}">
```

Comment:
There are other buttons `<button class="btn interact-bg gt-p-3...`, why this "dropdown btn" needs special styles (eg: `vertical-align: bottom;`) ?

Reply:
The `dropdown` class sets the vertical-align property for the dropdown itself, which resulted in misalignment of the download icon

Reply:
That's also related the side effect of global `.svg { vertical-align: ... }`

@silverwind I would suggest to make `.svg { vertical-align: middle; }` by default. What's the side effect of it?

Reply:
Otherwise, maybe we need to make the "ui button" and "ui dropdown" (even "flex-text....") have the same "vertical-align:" as ".svg"

Reply:
It is not `svg` but the button itself. I now created a combined class for this button row with `vertical-align: middle` for every single button.

Reply:
I tried on try.gitea.io, it seems that the `vertical-align` on ".svg" is directly related.

Since it's a longstanding problem, I guess it doesn't block

Reply:
Did you spot this kind of issue in other places? If so, I could take a look in another PR

Reply:
> That's also related to the side effect of global `.svg { vertical-align: ... }`
> 
> @silverwind I would suggest to make `.svg { vertical-align: middle; }` by default. What's the side effect of it?

Pretty sure this would cause regressions. FWIW, GitHub has `vertical-align: text-bottom` globally. I guess you can never make it right for all situations.